src/test/java/org/fcrepo/transform/transformations/LDPathTransformTest.java

```diff
@@ -20,32 +20,40 @@
 import static com.hp.hpl.jena.graph.Triple.create;
 import static java.util.stream.Stream.of;
 import static org.fcrepo.transform.transformations.LDPathTransform.CONFIGURATION_FOLDER;
-import static org.fcrepo.transform.transformations.LDPathTransform.getNodeTypeTransform;
+import static org.fcrepo.transform.transformations.LDPathTransform.getResourceTransform;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
-import static org.mockito.Mockito.RETURNS_DEEP_STUBS;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 import static org.mockito.MockitoAnnotations.initMocks;
+import static org.fcrepo.kernel.api.RdfLexicon.REPOSITORY_NAMESPACE;
 
 import java.io.ByteArrayInputStream;
 import java.io.InputStream;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
 import java.util.Map;
+import java.util.stream.Stream;
```

Comment:
very minor point, but `java.util.stream.Stream` should appear after `java.util.List`.